### DISCUSSION TITLE
Editing stats manually

### DISCUSSION THREAD

**Discussion by mouradbka** [focustimerhq/FocusTimer#597] [Ideas]
I would like to be able to edit the statistics of a day. 
Sometimes something go wrong or I forget to stop the timer and then the statistic are showing corrupted data. 

I would like to use this App to keep track of my working hours and these issues make it impossible to me. 

Thank you in advance

![pomodoro_error](https://user-images.githubusercontent.com/43741850/140869242-e97f90d1-416a-43e5-ae18-eb92b400a27f.jpg)
e!

**Comment by kamilprusko** [comment]
Thanks for feedback. I would like to address this while porting to gtk4.
